docs/api/cog.rst

```diff
@@ -0,0 +1,14 @@
+
+.. currentmodule:: pincer.cog
```

Comment:
sphinx accepts file level modules?

Reply:
i didn't even know there was another way to do this. It builds the docs right so its fine.